lib/s390base.pm

```diff
@@ -39,14 +39,19 @@ sub copy_testsuite {
     # fetch the testcase and common libraries from openQA server
     my $path        = data_url("s390x");
     my $script_path = "$path/$tc/$tc.tgz";
-    assert_script_run "mkdir -p ./$tc/ && cd ./$tc/ && rm -f $tc.tgz";
+    # remove previous files from SUT
+    assert_script_run("rm -rf /root/$tc/");
+    # get new version of test scripts
+    assert_script_run("mkdir -p ./$tc/");
+    assert_script_run("cd ./$tc/");
     assert_script_run "wget $script_path";
     assert_script_run "tar -xf $tc.tgz";
     my $commonsh_path = "$path/lib/common.tgz";
     assert_script_run "mkdir -p lib && cd lib && rm -f common.tgz";
     assert_script_run "wget $commonsh_path";
     assert_script_run "tar -xf common.tgz";
     assert_script_run "cd ..";
+    assert_script_run 'mkdir logs';
```

Comment:
I find that less readable and more difficult to debug when the test fails.

Reply:
> I find that less readable and more difficult to debug when the test fails.

But reduces the probability of the test failing due to a problem in the test setup, worst case scenario, you can prepare a script download it and call it :).

Reply:
I do not understand what you mean with 'test failing due to a problem in the test setup'.

Reply:
These commands, are setting up the test scenario, they're part of the test set up (via `copy_testsuite`. Only after the files are properly copied and in place at this point, the actual test, is not executed yet, only after ` $self->execute_script($blah)`, The actual tests are being ran.

So failures in the test setup, are not relevant other than to know, that the setup failed (Hence why I would push for a single command being executed, rather than multiple. As it's also easier to follow from the autoinst-log.txt perspective